### ISSUE TITLE
Exposed Host Ports should return Host IP in output

### ISSUE THREAD

**Issue by BenHall** [ContainerSolutions/minimesos#252]
When using the _--exposedHostPorts__ option, does it make more sense to return the Host IP address rather than the individual containers?

Currently the behaviour is:

```
minimesos up --exposedHostPorts                                                                     
export MINIMESOS_ZOOKEEPER=zk://172.18.0.3:2181                                                         
export MINIMESOS_MASTER=http://172.18.0.4:5050                                                          
export MINIMESOS_MARATHON=http://172.18.0.5:8080  
```

What I expected to see:

```
minimesos up --exposedHostPorts                                                                     
export MINIMESOS_ZOOKEEPER=zk://172.17.0.17:2181                                                         
export MINIMESOS_MASTER=http://172.17.0.17:5050                                                          
export MINIMESOS_MARATHON=http://172.17.0.17:8080  
```

When _--exposedHostPorts_ is not set then the existing behaviour should be preserved.

**Comment by sadovnikov** [comment]
Let's retest after #241 is merged. I believe I came across this and just resolved in that branch

**Comment by frankscholten** [comment]
https://github.com/ContainerSolutions/minimesos/pull/241 is merged

**Comment by sadovnikov** [comment]
Currently on `master` we get
```
vikmac:mini-mesos vik$ ./bin/minimesos up
# You are running minimesos on OS X, enabling --exposedHostPorts
export MINIMESOS_ZOOKEEPER=zk://192.168.99.104:2181
export MINIMESOS_MASTER=http://192.168.99.104:5050
export MINIMESOS_MARATHON=http://192.168.99.104:8080
```
or
```
vikmac:mini-mesos vik$ ./bin/minimesos up --exposedHostPorts
# You are running minimesos on OS X, enabling --exposedHostPorts
Failed to parse parameters. Can only specify option --exposedHostPorts once.
```

The first response is correct, but the second one is very wrong. Correcting...

**Comment by sadovnikov** [comment]
It's better now

```
vikmac:mini-mesos vik$ ./bin/minimesos up --exposedHostPorts
export MINIMESOS_ZOOKEEPER=zk://192.168.99.104:2181
export MINIMESOS_MASTER=http://192.168.99.104:5050
export MINIMESOS_MARATHON=http://192.168.99.104:8080

vikmac:mini-mesos vik$ ./bin/minimesos up 
# You are running minimesos on OS X, enabling --exposedHostPorts
export MINIMESOS_ZOOKEEPER=zk://192.168.99.104:2181
export MINIMESOS_MASTER=http://192.168.99.104:5050
export MINIMESOS_MARATHON=http://192.168.99.104:8080
```

Please, review and merge #265

**Comment by sadovnikov** [comment]
It is resolved within #265
